src/ace/managers/system.c

```diff
@@ -300,7 +302,11 @@ void systemCreate(void) {
 	WaitTOF(); // Wait for interlaced screen to finish
 
 	// get VBR location on 68010+ machine
-	// TODO VBR
+	if (SysBase->AttnFlags & AFF_68010)
+	{
```

Comment:
Brace is not codestyle compliant - move it past `if` condition bracket in same line